docs/reference/kernel_language.rst

```diff
@@ -1327,16 +1357,16 @@ To read a high-resolution timer from the device, HIP provides the following buil
 
   .. code:: cpp
 
-    clock_t clock()
-    long long int clock64()
+    long long int wall_clock64()
 
   The difference between the values that are returned represents the cycles used.
 
 * Returning the wall clock count at a constant frequency on the device:
 
   .. code:: cpp
 
-    long long int wall_clock64()
+    clock_t clock()
+    long long int clock64()
```

Comment:
long long int clock()
long long int clock64()
Should be before line 1362

long long int wall_clock64()
should be after line 1367

Reply:
Fixed.